src/amo/css/ScreenShots.scss

```diff
@@ -1,6 +1,10 @@
 @import "~core/css/inc/mixins";
 @import "~amo/css/inc/vars";
-$screenshot-height: 200px;
+
+//image-height has to be 20px smaller to ensure it doesn't overflow in the screenshots section.
+// See: https://github.com/mozilla/addons-frontend/issues/4887
+$image-height: $screenshot-height - 20px;
+$screenshot-height: 220px;
```

Comment:
@Vimal-Raghubir  - I just noticed this but screenshot-height should come before you use it on line 6